railties/lib/rails/commands/console/console_command.rb

```diff
@@ -38,6 +38,10 @@ def initialize(app, options = {})
 
       if @console == IRB
         IRB::WorkSpace.prepend(BacktraceCleaner)
+
+        if Rails.env == "production"
```

Comment:
Any reason to not use `Rails.env.production?` or even reverse the logic and set this only in development so all other environments (for example I have seen `staging` environment used often) will not be affected by completion?

Reply:
i would agree with the idea of reversing it and only enable it by default on development. 
in a perfect world, i would disable it for all environments and let the dev decide when to enable it, explicitly. either via `development.rb` or ENV var